ui/src/components/Header/Controller.tsx

```diff
@@ -14,13 +14,15 @@ import { DockerContainer, DockerImage } from '../../types';
 import { DownloadProgressDialog } from '../Feedback/DownloadProgressDialog';
 import { ProgressButton } from '../Feedback';
 
+const EXCLUDED_ERROR_TOAST = ['Successfully activated API key','Execution of "prepare_host"','DEBUG'];
```

Comment:
```suggestion
const EXCLUDED_ERROR_TOAST = ['Successfully activated API key', 'Execution of "prepare_host"', 'DEBUG'];
```

Again, would be great if the linter could catch this.. 👍